applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.hpp

```diff
@@ -304,6 +304,9 @@ namespace Kratos
          */		
 		void CalculateLocalNodalForces(const Vector& Bisectrix,const Vector& VectorDifference);
 
+	
+		std::string Info() const override; //fusseder needed for sensitivity analysis

```

Comment:
Remove this comments

Reply:
I don't understand this use of Info. I guess this is in line with @philbucher 's comment.